gui.js

```diff
@@ -2909,6 +2909,17 @@ var CGUI = function()
       }
     }
   };
+	
+	var presetIgnoreLetters = function (e)
+	{
+		if (!e)	var e = window.event;	
+		// Prevent letters (A-Z) and arrows to be used
+		if ((e.keyCode >= 65 && e.keyCode <= 90) || e.keyCode >= 37 || e.keyCode <= 40)
```

Comment:
Why limit to A-Z and arrow keys? This kind of logic is often problematic (how about "=", "(", "[", non-US characters/keyboards ...?).

What happens if we ignore all key presses?